backend/yarn.lock

```diff
@@ -2333,16 +2331,6 @@ fs.realpath@^1.0.0:
   resolved "https://registry.yarnpkg.com/fs.realpath/-/fs.realpath-1.0.0.tgz#1504ad2523158caa40db4a2787cb01411994ea4f"
   integrity sha1-FQStJSMVjKpA20onh8sBQRmU6k8=
 
-fsevents@^2.1.2:
-  version "2.2.1"
-  resolved "https://registry.yarnpkg.com/fsevents/-/fsevents-2.2.1.tgz#1fb02ded2036a8ac288d507a65962bd87b97628d"
-  integrity sha512-bTLYHSeC0UH/EFXS9KqWnXuOl/wHK5Z/d+ghd5AsFMYN7wIGkUCOJyzy88+wJKkZPGON8u4Z9f6U4FdgURE9qA==
-
-fsevents@~2.1.2:
-  version "2.1.3"
-  resolved "https://registry.yarnpkg.com/fsevents/-/fsevents-2.1.3.tgz#fb738703ae8d2f9fe900c33836ddebee8b97f23e"
-  integrity sha512-Auw9a4AxqWpa9GUfj370BMPzzyncfBABW8Mab7BGWBYDj4Isgq+cDKtx0i6u9jcX9pQDnswsaaOTgTmA5pEjuQ==
-
```

Comment:
I removed this because whenever I do yarn install, I always get a warning. I figured that `fsevents` is only for MacOS and I use windows. Is this important for MacOS?

Reply:
Oh interesting, what was the warning you were getting? Not sure how important for MacOS this is because I haven't come across it yet... Depending on the error it might help to clear your cache before running yarn install?

Reply:
I'd always come across this message. I tried researching this and found that its a matter of OS, but I can spend some time to research how to fix this properly
![image](https://user-images.githubusercontent.com/49849754/103193855-8775c700-4892-11eb-841c-5f64f6b8babb.png)

Reply:
Ohhh is this the same error? https://stackoverflow.com/questions/46929196/how-to-solve-npm-install-throwing-fsevents-warning-on-non-mac-os

Is the pic above from GA? Cause I'm thinking if it is and it's running Linux, the first answer might be helpful

Either way, I don't think there should be too much harm in just removing it for now. If we figure out that there's a need for this dependency when developing locally we can deal with it then?

Reply:
Ohh yah same error. I get the same thing for Windows as well. Yup 100% agree, sounds like good idea